libcudacxx/include/cuda/std/__bit/rotl.h

```
@@ -0,0 +1,190 @@
+    const auto __out_2 = __word_rot2 ? __tmp_0 : __tmp_2;
+    const auto __out_3 = __word_rot2 ? __tmp_1 : __tmp_3;
+
+    return (__uint128_t{__out_3} << 96) | (__uint128_t{__out_2} << 64) | (__uint128_t{__out_1} << 32) | __out_0;
+  }
+#  endif // _CCCL_HAS_INT128()
+  // NOLINTEND(bugprone-branch-clone)
+  else
+  {
+    return ::cuda::std::__rotl_impl_generic(__v, __cnt);
+  }
+  // NOLINTEND(bugprone-branch-clone)
```

Clang (14.0.6)

[error] unmatched 'NOLINTEND' comment without a previous 'NOLINTBEGIN' comment

(clang-tidy-nolint)